cdap-ui/app/cdap/components/PreviewData/index.tsx

```diff
@@ -25,6 +25,11 @@ import { messageTextStyle } from 'components/PreviewData/Table';
 import PreviewTableContainer from 'components/PreviewData/TableContainer';
 import withStyles, { WithStyles } from '@material-ui/core/styles/withStyles';
 import ThemeWrapper from 'components/ThemeWrapper';
+import T from 'i18n-react';
+import { extractErrorMessage } from 'components/Replicator/utilities';
```

Comment:
If the utility function is generic enough we need to pull it out. Its hard to track this change tomorrow if something changes in replicator.

Reply:
That's reasonable. Where would you suggest putting the utility function?

Reply:
Moved to services/helpers, as per our conversation